src/NetAnalyzers/CSharp/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA2014 | Reliability | Warning | CSharpDoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2014)
```

Comment:
This is an already shipped rule in .NET 5 release. It should be manually moved to shipped file.

Reply:
Also I would have expected build to fail as `CA2014` is in the core shipped analyzer releases file, but the rule is no longer in that project. You would have to delete the existing `CA2014` entry in that file.